pkg/sql/catalog/colinfo/result_columns.go

```diff
@@ -305,6 +305,7 @@ var TenantColumnsWithReplication = ResultColumns{
 	{Name: "source_tenant_name", Typ: types.String},
 	{Name: "source_cluster_uri", Typ: types.String},
 	{Name: "replication_job_id", Typ: types.Int},
```

Comment:
I chatted with @alicia-l2 yesterday and we were we would remove replication_job_id to make space for the new lag column. She also thought it would be more intuitive to order it as retained_time, replicated_time, replication_lag, cutover_time

Reply:
Done.

Reply:
Hrm. Are we sure about this?  For instance if you find your replication job PAUSED or FAILED, how confident are we that in this command we always show the relevant error output? I suppose `SELECT * FROM [SHOW JOBS] WHERE job_type = 'STREAM INGESTION'` isn't too hard. but it is a bit more than copy/pasting the job ID and doing `SHOW JOB`.

Reply:
> how confident are we that in this command we always show the relevant error output

This sounds like something _we_ should be on the hook to find/fix. Needing to go find the job ID in `show jobs` or the db console is _slightly_ annoying sure, but if it is just something that we would be doing when debugging a bug and then we go fix the status update code, that seems like the right trade for the end-user UX being that we show them just the things that are relevant _to them_, which is the replication status?

Reply:
One observation is that this table is _much_ easier to read when it fits on one line. Once it wraps it is much harder to line up which timestamp your'e looking at with the headers. The job_id's cost, in line width and noise, to most users seems higher than the benefit give our not-job-centric UX here, so if it is just there for debugging by CRL engineers, I think we can say we're on the hook for finding it the slightly harder way to reclaim the line width.

Reply:
OK. I'll follow up next week with a test that makes sure we get the job error in the status output. I feel OK arguing any fix we need there is worth backporting.

Reply:
(... in the `with replication status` version) and rename `data_state` to `status` and remove `service_mode` from this version while we're at it?

Reply:
> Should we move the status column all the way to the right while we're here in case there is a long error in it?

I think we should.  We'll need to reach out to preview users who have queries against this already, but that is fine.

> (... in the with replication status version) and rename data_state to status and remove service_mode from this version while we're at it?

I think in the original command we were thinking that _WITH_ would only _add_ columns.  But, if we are making breaking changes we shouldn't be shy.

Reply:
just pushed up another commit. if CI is pleased, I'll squash and merge.